client/src/components/FeedbackPage/FeedbackFormSection/FeedbackFormSection.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { FunctionComponent, useState, HTMLInputTypeAttribute } from 'react';
+
+// Types
+import { InputElement } from 'types';
+
+// Components
+import Input from 'components/Input/Input';
+
+// Styles
+import { Container, Flex } from 'styles/global';
+import * as S from '../FeedbackPage.styles';
+
+enum FeedbackFormElement {
+  name = 'name',
+  rating = 'rating',
+  reasons = 'reasons',
+  suggestions = 'suggestions',
+  recommend = 'recommend',
+  more = 'more'
+}
+
+type FeedbackForm = {
+  [key in FeedbackFormElement]: {
+    label: string;
+    elementType: InputElement;
+    elementConfig: {
+      type?: HTMLInputTypeAttribute;
+      placeholder?: string;
+      rows?: number;
+      options?: Record<string, unknown>;
+    };
+  };
+};
+
+const FeedbackFormSection: FunctionComponent = () => {
+  const [feedbackForm, setFeebackForm] = useState<FeedbackForm>({
+    name: {
+      label: '1. What is your name?',
+      elementType: 'input',
+      elementConfig: {
+        type: 'input',
+        placeholder: 'Please tell us your full name...'
+      }
+    },
+    rating: {
+      label: '2. Rate your experience with our app.',
+      elementType: 'rating',
+      elementConfig: {
+        type: 'radio',
+        options: {
+          'rating-1': 1,
+          'rating-2': 2,
+          'rating-3': 3,
+          'rating-4': 4,
+          'rating-5': 5
+        }
+      }
+    },
+    reasons: {
+      label: '3. Tell us your reasons for giving this score.',
+      elementType: 'textarea',
+      elementConfig: {
+        placeholder: 'Please state your reasons here...',
+        rows: 3
+      }
+    },
+    suggestions: {
+      label: '4. Anything that can be improved?',
+      elementType: 'textarea',
+      elementConfig: {
+        placeholder: 'Please tell us what to improve, if any...',
+        rows: 3
+      }
+    },
+    recommend: {
+      label: '5. Would you recommend this app to someone else?',
+      elementType: 'radio',
+      elementConfig: {
+        type: 'radio',
+        options: {
+          'option-1': 'Yes',
+          'option-2': 'No'
+        }
+      }
+    },
+    more: {
+      label: '6. Care to share more?',
+      elementType: 'textarea',
+      elementConfig: {
+        placeholder: 'Anything more?...',
+        rows: 3
+      }
+    }
+  });
```

Comment:
1. this object is not updated during component lifecycle, so no need to use state here. please move this variable out of component body
2. you can use enum values instead of property names `more` => `[FeedbackFormElement.more]`

Reply:
1. I will expand it with input value, valid state, etc. which will be stored in this object.

Reply:
when you'll start adding input values, just create another state (it can be with the same structure` {<input name>: <input value> }`). state should be used for dynamic values only, hardcoded text should be stored as separate variable